pkg/render/node.go

```diff
@@ -1296,6 +1296,9 @@ func (c *nodeComponent) nodeLivenessReadinessProbes() (*v1.Probe, *v1.Probe) {
 	}
 	rp := &v1.Probe{
 		Handler: v1.Handler{Exec: &v1.ExecAction{Command: readinessCmd}},
+		//period is default but timeout is half to account for some slow machines
```

Comment:
I'd prefer this explain why we're not using the default but with a caveat of keeping it less than the period. Maybe something like
```
Set the TimeoutSeconds greater than the default of 1 to allow additional time on loaded nodes. Keep this time less than the PeriodSeconds.
```